Phase1/test/FilterHandlerTest.java

```diff
@@ -0,0 +1,29 @@
+package test;
+
+import src.*;
+import java.util.*;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class FilterHandlerTest {
+    private InvertedIndex invertedIndex = new InvertedIndex();
```

Comment:
When your are using methods from other classes (like filter method in plusFilter, minusFilter, etc..) you should mock the object and subtle these methods, for example imagine some situation that filter method from PlusFilter doesn't work as well so this test will be failed too and make debugging code annoying for you.